### PR TITLE
Included the new R Help button on all calculator keyboards

### LINKED ISSUE #8031: Include the new R Help button on all calculator keyboards
@derekagorhom I hope this is another quick improvement for you?

Could you include the R Help button with the down arrow on each keyboard - even when there is just a single package.  It enables the package name to be given.

1) Maths 1 package: stats 
2) Logical - 2 packages, namely base and dplyr.
Could you also make the keyboard neater.  I suggest moving " " and NA to the top 2 rows and making the first 3 rows of keys line up, with 6 keys in each.  Then change the `%%` key to `mod (%%)`  and the `%/%` to `div (%/%)`.
Then there are 4 keys on those rows.
3) summary - already done
4) Text strings: `stringr` is main one so first then `base  `
5) factor - I package, forcats
6) probability stats is first, then base
7) dates lubridate first, then hms
8) transform - already done
9) wakefield wakefield
10) circular - 1 package circular
Can you also move the label on that odd key down, so it lines up with the others.
11) hydro  - just hydroGOF
12) lists (when you have it.  base, then stats then statip 
13) complex - already done
14) integer - already done

### PR DESCRIPTION
**Pull request by derekagorhom**
fixes #8031 
this PR is ready for review
Screenshot (515)

I also added the `ContextMenuStrip` for the `List keyboard` (see above) so that it can be added when the keyboard is ready

### CODE REVIEW

**Comment by rdstern** [comment]
@derekagorhom this looks great.  It all seems to work fine.  I am just being picky here, with the changes in the logical keyboard.  

image

It already looks much better:
a) could you add a space in div (%/%)
b) could you line up the first 3 rows even more.  So the 3rd row lines with the first two.
c) And the first 3 rows (where there are 6 keys) line with the lower rows - so the end of the 3rd key in the top lines is the same as the end of the 2nd key in the bottom lines.

Then it can be merged.

**Review by rdstern — DISMISSED** [review]
Excellent

File: instat/ucrCalculator.vb [hunk 1]
```diff
@@ -4611,6 +4611,182 @@ Public Class ucrCalculator
...
 
+    Private Sub cmdWakefieldHelp_Click(sender As Object, e As EventArgs) Handles cmdWakefieldHelp.Click
+        CalculationsOptions()
+        If ucrInputCalOptions.GetText = "Wakefield" Then
+            strPackageName = "wakefield"
+        End If
+        OpenHelpPage()
+    End Sub
+
+    Private Sub WakefieldToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles WakefieldToolStripMenuItem.Click
```
Comment (lloyddewit) [inline instat/ucrCalculator.vb:4622]: This sub looks the same as the sub above. Could the sub above also handle the `WakefieldToolStripMenuItem.Click` event?

This comment also applies to most of the new subs below. Could some of the subs below be combined?

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@derekagorhom Thanks, looks good, just one comment

File: instat/ucrCalculator.vb (hunk 1 above)
Comment (derekagorhom) [reply instat/ucrCalculator.vb:4606]: @lloyddewit i have made the changes, can you review it again

File: instat/ucrCalculator.vb [hunk 2]
```diff
@@ -1504,7 +1504,7 @@ Public Class ucrCalculator
                                    iCallType:=2, bSeparateThread:=False, bUpdateGrids:=False)
     End Sub
 
-    Private Sub cmdHelp_Click(sender As Object, e As EventArgs) Handles cmdRHelp.Click, cmdHydroHelp.Click, cmdCircularHelp.Click, cmdWakefieldHelp.Click, cmdMathsHelp.Click, cmdLogicalHelp.Click, cmdProbRHelp.Click, cmdStringRHelp.Click
+    Private Sub cmdHelp_Click(sender As Object, e As EventArgs)
```
Comment (lloyddewit) [inline instat/ucrCalculator.vb:1507]: This sub doesn't seem to handle any events. Is it still needed or could it be deleted?

File: instat/ucrCalculator.vb [hunk 3]
```diff
@@ -4328,11 +4328,7 @@ Public Class ucrCalculator
     End Sub
 
     Private Sub GmpToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles GmpToolStripMenuItem.Click
```
Comment (lloyddewit) [inline instat/ucrCalculator.vb:4330]: If we add this event to the `cmdIntegerRHelp()` sub, then could we delete this sub?
```
    Private Sub cmdIntegerRHelp_Click(sender As Object, e As EventArgs) Handles cmdIntegerRHelp.Click, GmpToolStripMenuItem.Click
```

File: instat/ucrCalculator.vb (hunk 1 above)
Comment (lloyddewit) [reply instat/ucrCalculator.vb:4606]: Sorry if I wasn't clear. I was suggesting to add the `WakefieldToolStripMenuItem.Click` event to the sub above (see suggested change above).

File: instat/ucrCalculator.vb [hunk 4]
```diff
@@ -4604,13 +4592,149 @@ Public Class ucrCalculator
     End Sub
 
     Private Sub ComplexBaseToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles ComplexBaseToolStripMenuItem.Click
+        cmdComplexRHelp.PerformClick()
+    End Sub
+
+    Private Sub cmdWakefieldHelp_Click(sender As Object, e As EventArgs) Handles cmdWakefieldHelp.Click
```
Comment (lloyddewit) [inline instat/ucrCalculator.vb:4598]: Would this work?
```suggestion
    Private Sub cmdWakefieldHelp_Click(sender As Object, e As EventArgs) Handles cmdWakefieldHelp.Click, WakefieldToolStripMenuItem.Click
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Comment by derekagorhom** [comment]
@lloyddewit i have made the changes

File: instat/ucrCalculator.vb [hunk 5]
```diff
@@ -4540,11 +4520,7 @@ Public Class ucrCalculator
     End Sub
 
     Private Sub BaseToolStripMenuItem1_Click(sender As Object, e As EventArgs) Handles BaseToolStripMenuItem1.Click
```
Comment (lloyddewit) [inline instat/ucrCalculator.vb:4522]: can we delete this?

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@derekagorhom Thanks, just one open question and then I think we can merge

File: instat/ucrCalculator.vb (hunk 5 above)
Comment (derekagorhom) [reply instat/ucrCalculator.vb]: Sorry i missed that, it has been removed

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern If you can retest/approve, then we can merge, thanks

**Review by rdstern — APPROVED** [review]

**Pull request merged**